gatsby/content/blog/2022/04/2022-04-05-synapse-1.56.mdx

```diff
@@ -51,9 +51,8 @@ at the time.
 Now that spaces have been out of beta for some time, and have shown to be a very
 useful feature to the ecosystem, we decided it was time to retire support for
 groups from Synapse, after more than 4 years of good and loyal service. This
-release of Synapse deprecates the feature and disables it by default, with a
-plan to completely remove it in Synapse 1.58 (which is expected to be released
-next month).
+release of Synapse deprecates the feature, with a plan to completely remove it
+in Synapse 1.58 (which is expected to be released next month).
```

Comment:
```suggestion
release of Synapse deprecates the feature, with a plan to disable it by default
in Synapse 1.58 (which is expected to be released next month).
```

Reply:
For cross-linking, see https://github.com/matrix-org/synapse/pull/12344 which will land in v1.58.0.